From 3113740befdc881374240b0292d5deda4cdb60ac Mon Sep 17 00:00:00 2001
From: M. Ranganathan <mranga@sipxpbx.example.com>
Date: Sat, 28 Jun 2008 23:47:55 -0400
Subject: [PATCH] Changes as a result of interop testing with bandwidth.com configuration rquirements:

Allow global port to be specified. Allow registration interval to be specified.
---
 sipXconfig/neoconf/etc/commserver/siptrunk.xml |    9 ++++++++-
 1 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/sipXconfig/neoconf/etc/commserver/siptrunk.xml b/sipXconfig/neoconf/etc/commserver/siptrunk.xml
index 4aeecdb..3cb6448 100644
--- a/sipXconfig/neoconf/etc/commserver/siptrunk.xml
+++ b/sipXconfig/neoconf/etc/commserver/siptrunk.xml
@@ -44,7 +44,7 @@
     <setting name="user-name">
       <label>User</label>
       <type>
-        <string />
+        <string/>
       </type>
       <description>The user name attached to the account.</description>
     </setting>
@@ -86,6 +86,13 @@
       <value>600</value>
       <description>Registration Interval (seconds)</description>
     </setting>
+    <setting name="registration-interval" advanced="yes">
+      <type>
+        <integer/>
+      </type>
+      <value>600</value>
+      <description>Registration Interval (seconds) </description>
+    </setting>
     <setting name="use-registration-for-caller-id" advanced="yes">
       <label>Use Registration for caller-ID</label>
       <type refid="switch"></type>
-- 
1.5.4.1

